src/core/index.js

```diff
@@ -49,14 +49,14 @@ module.exports = function SwaggerUI(opts) {
     defaultModelExpandDepth: 1,
     defaultModelsExpandDepth: 1,
     showExtensions: false,
+    tagsSorter: "alpha",
     supportedSubmitMethods: [
       "get",
       "put",
       "post",
       "delete",
       "options",
       "head",
-      "patch",
```

Comment:
Was this intentional? I don't see it in the (amazing!) description in the pull request.